lib/src/soloud.dart

```diff
@@ -358,6 +364,32 @@ interface class SoLoud {
     }
   }
 
+  /// Changes the output audio device to the one specified in the [newDevice].
+  /// If [newDevice] is not provided, the default OS device will be used.
+  ///
+  /// Note: Android and Web, only support one output device which is the
+  /// default device (iOS and MacOS?).
+  ///
+  /// Throws [SoLoudNotInitializedException] if the engine is not initialized.
+  PlayerErrors changeDevice({PlaybackDevice? newDevice}) {
```

Comment:
If I remember correctly, I think we stopped returning `PlayerErrors` and instead just throw Dart exceptions. In this case, I think it would make more sense to throw, and then just return `void` (if the change is instantaneous) or `Future<void>` (if it's async).

Reply:
Fixed and added `SoLoudNoPlaybackDevicesFoundCppException`.